### PR TITLE
Signpost to API reference for ServiceAccount

### LINKED ISSUE #45747: Link to API reference for ServiceAccount
**This is a Feature Request**

**What would you like to be added**
Update the [front matter](https://gohugo.io/content-management/front-matter/) for https://kubernetes.io/docs/concepts/security/service-accounts/ to link to the API reference for that page.

You can find the API group and kind for ServiceAccount if you start from https://kubernetes.io/docs/concepts/security/service-accounts/#what-s-next, and you can look at https://github.com/kubernetes/website/pull/45746 to see a PR that does a similar thing for NetworkPolicy. Note that ServiceAccount is in the core API group so the API group string won't contain a `/` (unlike NetworkPolicy).

If you're not sure what the change should look like, visit https://github.com/kubernetes/website/pull/45746/files

**Why is this needed**
This change helps readers find the API reference documentation quickly.

**Comments**
/language en
/wg api-expression
/help

### PR DESCRIPTION
**Pull request by dev-johnn**
Link to API reference for ServiceAccount as requested in #45747
 
Closes #45747

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Signpost to API reference for ServiceAccount

File: content/en/docs/concepts/security/service-accounts.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title: Service Accounts
 description: >
   Learn about ServiceAccount objects in Kubernetes.
+api_metadata:
+- apiVersion: "k8s.io/api/core/v1"
```
Comment (sftim) [inline content/en/docs/concepts/security/service-accounts.md:6]: Hi; this should actually be:
```suggestion
- apiVersion: "v1"
```

**Review by sftim — COMMENTED** [review]
For the core API group, there's no `/` within the API version.

**Review by dipesh-rawat — COMMENTED** [review]
The change looks good. Thanks!

[Preview Page](https://deploy-preview-45750--kubernetes-io-main-staging.netlify.app/docs/concepts/security/service-accounts/)

/lgtm
/approve

**Pull request merged**